socket.c

```diff
@@ -289,6 +289,7 @@ int cSatipSocket::ReadMulti(unsigned char *bufferAddrP, unsigned int *elementRec
      error("%s Invalid parameter(s)", __PRETTY_FUNCTION__);
      return -1;
      }
+#if defined(__GLIBC__)
 #if defined(__GLIBC_PREREQ) && __GLIBC_PREREQ(2,12)
```

Comment:
This will not work as a one liner with musl-clib. It has to be a two-liner.
The wohle line get evaluated, and because __GLIBC_PREREQ isn't definded, it result in a syntax error. Booth statements get evaluated, regardless if the first one was "false"